src/index.tsx

```diff
@@ -2,4 +2,4 @@ import ReactDOM from 'react-dom';
 import 'bulma/css/bulma.css';
 import { App } from './App';
 
-ReactDOM.render(<App />, document.getElementById('root'));
+ReactDOM.render(<App delay={500} />, document.getElementById('root'));
```

Comment:
let's increase the default delay, and better to remove delay as a prop and just add it as a constant in app.tsx